load_sample_data.sh

```diff
@@ -102,10 +102,14 @@ just load-test-data "image"
 sleep 2
 
 # Ingest and index the data
-just ingest-upstream "audio"
+just ingest-upstream "audio" "init"
+just wait-for-index "audio-init"
+just point-alias "audio" "init" "audio"
 just wait-for-index "audio"
 
-just ingest-upstream "image"
+just ingest-upstream "image" "init"
+just wait-for-index "image-init"
```

Comment:
`just wait-for-index ...` throw an error for audio and image to me: `sh: timeout: command not found`.

Which causes an error at the search list endpoint:

```python
NotFoundError(404, 'index_not_found_exception', 'no such index [audio]', audio, index_or_alias)
```

Unless I point the alias manually, then I get no audio results 😓

Reply:
Ah, `timeout` has to be installed separately I suppose. I must have got that installed as a dependency of some other package.

<img width="712" alt="Screenshot 2022-06-30 at 7 51 58 AM" src="https://user-images.githubusercontent.com/16580576/176588997-4846a818-08d3-4c2c-916c-4199f7fc9284.png">
 
Could you try installing [`coreutils` using Homebrew](https://formulae.brew.sh/formula/coreutils#default)? If that fixes the problem, I'll update the docs with this prerequisite, or change the timeout script to use Python as the base language.

Reply:
It worked! Though it would be nice if we could spare that dependency. Thanks for looking at this Dhruv!